starknet_py/contract.py

```diff
@@ -3,14 +3,15 @@
 import dataclasses
 import json
 import warnings
+from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from functools import cached_property
-from typing import Dict, List, Optional, Tuple, TypeVar, Union
+from typing import Dict, List, Optional, TypeVar, Union
 
 from marshmallow import ValidationError
 
-from starknet_py.abi.model import Abi
-from starknet_py.abi.parser import AbiParser
+from starknet_py.abi import Abi as AbiV0
+from starknet_py.abi import AbiParser as AbiV0Parser
```

Comment:
Nit: Maybe move it to `starknet_py.abi.v0.model` for consistency with other abis?
Might be done in a separate PR if that would clutter diff.

Reply:
Thanks! Will be done in #1270